### PR TITLE
New package: bmap-tools-3.5

### PR DESCRIPTION
**Pull request by anjandev**

### CODE REVIEW

File: srcpkgs/bmap-tools/template [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Template file for 'bmap-tools'
+pkgname=bmap-tools
+version=3.5
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
```
Comment (xtraeme) [inline srcpkgs/bmap-tools/template:6]: missing:

- `archs=noarch`
- `depends="python3-setuptools"`

Comment (anjandev) [reply srcpkgs/bmap-tools/template:7]: fixed. Thanks

File: srcpkgs/bmap-tools/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'bmap-tools'
+pkgname=bmap-tools
+version=3.5
```
Comment (abenson) [inline srcpkgs/bmap-tools/template:3]: Update this to 3.6

File: srcpkgs/bmap-tools/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+revision=1
+archs=noarch
```
Comment (abenson) [inline srcpkgs/bmap-tools/template:5]: drop `noarch`

**Pull request closed without merging**